### ISSUE TITLE
kleines CSS-Problem bei Video mit Auflösung "mqdefault" im Social Media Footer

### ISSUE THREAD

**Issue by Albager** [RRZE-Webteam/rrze-video#18]
Im Social Media Footer habe ich ein Video platziert mit folgenden Einstellungen:
![image](https://user-images.githubusercontent.com/11389126/39618533-78c92a80-4f84-11e8-8694-8df2e8be7c3a.png)

Im Social Media Footer wird das Video in der mobilen Ansicht jedoch nicht wie die anderen Social Media Footer-Widgets zentriert, außerdem fehlt die margin oben und unten:
![image](https://user-images.githubusercontent.com/11389126/39618564-9a2cc4fc-4f84-11e8-816a-5c212eae56ea.png)

**Comment by xwolfde** [comment]
Prüfen ob das Problem Teil des Themes oder des Video-Plugins ist.
Wenn Video-Plugin, dann hier lösen.

**Comment by Albager** [comment]
Mittlerweile sieht das Video im Social Media Footer übrigens so aus (vgl. Play-Button):

![image](https://user-images.githubusercontent.com/11389126/47565608-5f445380-d929-11e8-9334-21d515dcc5a3.png)

**Comment by xwolfde** [comment]
sollte gefixt sein.

**Comment by Albager** [comment]
Hmmmmm - so richtig schön sieht es leider immer noch nicht aus :smile: Der Button ist OK, aber das Video ist nicht richtig zentriert:
![image](https://user-images.githubusercontent.com/11389126/49436000-3d57af80-f7b8-11e8-840c-cb068bb086f8.png)

**Comment by xwolfde** [comment]
:( 

Ist das bei euch ein YT Video? Und wenn ja, welches Format habt ihr da eingestellt? Bei meinen Test hatte ich bei meinen Videos aus YT nicht dieses Format.

**Comment by Albager** [comment]
Das Video ist ein youtube-Video in unserer "Videothek":
![image](https://user-images.githubusercontent.com/11389126/49441408-a80fe780-f7c6-11e8-87b5-d0a5eb332097.png)

Im Social-Media-Footer ist es folgendermaßen eingebunden:
![image](https://user-images.githubusercontent.com/11389126/49441451-c83fa680-f7c6-11e8-898a-358271ada5db.png)

Soll ich es mal "direkt" mit der YT-URL versuchen?

**Comment by xwolfde** [comment]
Die URL please...

**Comment by Albager** [comment]
Ja, das ist klar... :smile: Dennoch: Wenn ich versuche, bei "Von URL einfügen" einen Youtube-Stream in den Social Media Footer (oder sonstwo) einzufügen, wird dieser nicht erkannt. Wär' das ein neuer Issue?

**Comment by xwolfde** [comment]
Das müsstest du bei Wordpress selbst als Issue eintüten :)  Das ist eine Funktion der Medientheak, die IMHO nichts mit dem Plugin oder dem Theme an sich zu tun hat.
